x-pack/plugins/apm/public/application/uxApp.tsx

```diff
@@ -71,7 +72,11 @@ function UxApp() {
         darkMode,
       })}
     >
-      <div data-test-subj="csmMainContainer" role="main">
+      <div
+        className={APP_WRAPPER_CLASS}
+        data-test-subj="csmMainContainer"
+        role="main"
+      >
```

Comment:
@cchaos Do you think we can cherry-pick these changes for PR target for fixing https://github.com/elastic/kibana/issues/102311 in 7.15? The layout issue would be considered a bug for the current release.

Reply:
Yeah, should be no problem. Will you take care of it?

Reply:
@cchaos Yup, np

Reply:
Opened https://github.com/elastic/kibana/pull/110058 to fix this for UX and Uptime